### PR TITLE
Made barrier items able to be dragged, carried and also able to be put in cargo.

### PR DESCRIPTION
**Pull request by ColdEvul**
Made following items able to be dragged carried and put in to cargo: 
RoadCone_F
<s>RoadCone_L_F</s>
RoadBarrier_F
<s>RoadBarrier_small_F</s>

EDIT removed the classnames that had issues.

### CODE REVIEW

File: addons/cargo/CfgEventHandlers.hpp [hunk 1]
```diff
@@ -88,4 +88,14 @@ class Extended_InitPost_EventHandlers {
             init = QUOTE(_this call DFUNC(initObject));
         };
     };
+    class RoadCone_F {
+        class ADDON {
+            init = QUOTE(_this call DFUNC(initObject));
+        };
+    };
+    class RoadCone_L_F {
```
Comment (jonpas) [inline addons/cargo/CfgEventHandlers.hpp]: This class is insie `ReammoBox_F` class which already has XEH in this file, no need to have it twice.

File: addons/cargo/CfgVehicles.hpp [hunk 2]
```diff
@@ -426,4 +426,45 @@ class CfgVehicles {
             };
         };
    };
+   //Objects
+	class FloatingStructure_F; // External class reference
+    class RoadCone_F: thingX {
+        GVAR(size) = 1; // 1 = small, 2 = large
+        GVAR(canLoad) = 1;
+		XEH_ENABLED;
```
Comment (jonpas) [inline addons/cargo/CfgVehicles.hpp]: Indentation

File: addons/cargo/CfgVehicles.hpp [hunk 3]
```diff
@@ -426,4 +426,45 @@ class CfgVehicles {
...
+                statement = "";
+                icon = "\a3\ui_f\data\IGUI\Cfg\Actions\eject_ca.paa";
+                selection = "";
+            };
+        };
+    };
+    class RoadCone_L_F: FloatingStructure_F {
+        GVAR(size) = 1; // 1 = small, 2 = large
+        GVAR(canLoad) = 1;
+		XEH_ENABLED;
```
Comment (jonpas) [inline addons/cargo/CfgVehicles.hpp]: Indentation

File: addons/cargo/CfgVehicles.hpp [hunk 4]
```diff
@@ -426,4 +426,45 @@ class CfgVehicles {
             };
         };
    };
+   //Objects
+	class FloatingStructure_F; // External class reference
```
Comment (jonpas) [inline addons/cargo/CfgVehicles.hpp]: Indentation

File: addons/dragging/CfgEventHandlers.hpp [hunk 5]
```diff
@@ -32,6 +32,16 @@ class Extended_Init_EventHandlers {
             init = QUOTE(_this call DFUNC(initObject));
         };
     };
+    class RoadCone_F {
+        class ADDON {
+            init = QUOTE(_this call DFUNC(initObject));
+        };
+    };
+    class RoadCone_L_F {
```
Comment (jonpas) [inline addons/dragging/CfgEventHandlers.hpp]: This class is insie `ReammoBox_F` class which already has XEH in this file, no need to have it twice.

**Comment by jonpas** [comment]
Add newline between classes for readability.

File: addons/cargo/CfgVehicles.hpp (hunk 2 above)
Comment (ColdEvul) [inline addons/cargo/CfgVehicles.hpp]: That was not present in my original file

Comment (jonpas) [inline addons/cargo/CfgVehicles.hpp]: What do you mean? All I want to say is that it's not indented correctly, you are using a tab here, it should be 4 spaces as per ACE3 standards.

File: addons/dragging/CfgEventHandlers.hpp (hunk 5 above)
Comment (ColdEvul) [inline addons/dragging/CfgEventHandlers.hpp]: Removed

File: addons/cargo/CfgVehicles.hpp (hunk 3 above)
Comment (ColdEvul) [inline addons/cargo/CfgVehicles.hpp]: Fixed

File: addons/dragging/CfgEventHandlers.hpp (hunk 5 above)
Comment (ColdEvul) [inline addons/dragging/CfgEventHandlers.hpp]: And not 100% on what it means :)

**Comment by ColdEvul** [comment]
Weird issue suddenly appeared for me.

Both RoadCone_L_F (and RoadBarrier_small_F that is the same) can't be unloaded from a vehicle inventory. While RoadCone_F (and RoadBarrier_F that is the same) can.

I've also experimented by adding Land_CncBarrier_F and Land_CncBarrier_stripes_F as a movable item. "That is only draggable due to weight" but when unloading them (loading them "cost" 2 spaces cause they are heavy and large) they become inadvisable. until they are interacted with.

File: addons/dragging/CfgEventHandlers.hpp (hunk 5 above)
Comment (ColdEvul) [inline addons/dragging/CfgEventHandlers.hpp]: "RoadCone_F" and "RoadCone_L_F" is not part of "ReammoBox_F" its a part of "thingX".

File: addons/cargo/CfgEventHandlers.hpp (hunk 1 above)
Comment (ColdEvul) [inline addons/cargo/CfgEventHandlers.hpp]: <code>RoadCone_F</code> and <code>RoadCone_L_F</code> is not part of <code>ReammoBox_F</code> its a part of <code>thingX</code>.

**Comment by ColdEvul** [comment]
Remade it and made a new pullrequest:
https://github.com/acemod/ACE3/pull/2797

**Pull request closed without merging**